MovingBlocks/Terasology
# build(idea): base run configs on Gradle

### Contains

 As a result of looking into #5256 I propose to make our run configurations use Gradle instead of IntelliJ's built-in build system.

![grafik](https://github.com/MovingBlocks/Terasology/assets/1448874/f7bab2dd-6c83-4c48-a794-822684aae200)

### How to test

Try out the different configurations, and observe wheter anything changes drastically from the previous run configuration behavior.

Let me know whether you are missing a configuration you are using often and that was removed in this PR.

### Outstanding before merging

- [ ] is this setup feasible for attaching the debugger
- [ ] update the documentation accordingly
